### PR TITLE
Add id parameter to clay-chart templates | Fixes #1115

### LINKED ISSUE #1115: [clay-charts] Add 'id' parameter to template
Currently the .soy template in clay-charts does not allow receiving an `id` parameter and we need to fix that.

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

File: packages/clay-charts/src/Chart.soy [hunk 1]
```diff
@@ -5,8 +5,16 @@
  */
 {template .render}
 	{@param? _loading:  bool}
+	{@param? id: string}
 
-	<div class="metal-chart-container">
+  {let $attrs kind="attributes"}
```
Comment (jbalsas) [inline packages/clay-charts/src/Chart.soy]: Looks like a wrong indentation here... 🤔

File: packages/clay-charts/src/Geomap.soy [hunk 2]
```diff
@@ -6,5 +6,23 @@
 {template .render}
 	{@param? _height: string}
 	{@param? _width: string}
-	<div style="width:{$_width};height:{$_height}"></div>
+	{@param? id: string}
+
+  {let $attributes kind="attributes"}
```
Comment (jbalsas) [inline packages/clay-charts/src/Geomap.soy]: Wrong indentation?

Comment (julien) [reply packages/clay-charts/src/Geomap.soy]: Yes, I'll fix that
Thanks

File: packages/clay-charts/src/Chart.soy (hunk 1 above)
Comment (julien) [reply packages/clay-charts/src/Chart.soy]: Yes

**Pull request merged**